test/base/src/androidTest/kotlin/io/realm/test/shared/ImportTests.kt

```diff
@@ -199,12 +198,10 @@ class ImportTests {
         }
 
         val importedRoot = realm.writeBlocking {
-            copyToRealm(unmanaged)
+            assertFailsWithMessage<IllegalArgumentException>("Cannot import an outdated object") {
+                copyToRealm(unmanaged)
```

Comment:
Having the variable named `unmanaged` here is a bit weird and is messing with my head.. It is correct, but maybe calling it something like `outdatedContainer` or `unmanagedWithOldManagedObj` would make it a little more clear.